practice1.cpp

```diff
@@ -0,0 +1,178 @@
+        (&footbolist1)->role == (&footbolist2)->role &&
+        (&footbolist1)->age == (&footbolist2)->age &&
+        (&footbolist1)->games_count == (&footbolist2)->games_count &&
+        (&footbolist1)->goal_count == (&footbolist2)->goal_count;
+}
+int FindFootbolist(Footbolist* footbolist, int n, Footbolist footbolist0) {
+    for (int i = 0; i < n; i++) if (SameFootbolist(footbolist[i], footbolist0)) return i;
+    return -1;
+}
+Footbolist SetFootbolist() {
+    Footbolist footbolist;
+    cout << "Footbolist's surname: ";
+    cin >> (&footbolist)->surname;
+    cout << "Footbolist's role: ";
+    cin >> (&footbolist)->role;
+    cout << "Footbolist's age: ";
+    cin >> (&footbolist)->age;
+    cout << "Footbolist's games count: ";
+    cin >> (&footbolist)->games_count;
+    cout << "Footbolist's goal count: ";
+    cin >> (&footbolist)->goal_count;
+    return footbolist;
+}
+/////////////////////////////////////////////////////////////////////
+Footbolist * ImportFootbolists(fstream* FFile, string filename, int * n){
+    Footbolist* footbolist;
+    (*FFile).open(filename);
+    *FFile >> *n;
+    footbolist = new Footbolist[*n];
+    for (int i = 0; i < *n; i++) {
+        *FFile >> (footbolist + i)->surname >> (footbolist + i)->role >>
+            (footbolist + i)->age >> (footbolist + i)->games_count >> 
+            (footbolist + i)->goal_count;
+    }
+    (*FFile).close();
+    return footbolist;
+}
+void ExportFootbolists(fstream* FFile, string filename, Footbolist* footbolist, int n) {
+    (*FFile).open(filename, ofstream::out | ofstream::trunc);
+    (*FFile) << n << " ";
+    for (int i = 0; i < n; i++) {
+        (*FFile) << (footbolist + i)->surname << " " << (footbolist + i)->role << " " <<
+            (footbolist + i)->age << " " << (footbolist + i)->games_count << " " <<
+            (footbolist + i)->goal_count;
+    }
+    (*FFile).close();
+}
+/////////////////////////////////////////////////////////////////////
+Footbolist* AddFootbolist(Footbolist* footbolist, int* n, Footbolist footbolist0) {
+    Footbolist* newfootbolist = new Footbolist[((*n)+1)];
+    for (int i = 0; i < *n; i++) {
+        *(newfootbolist + i) = *(footbolist + i);
+    }
+    *(newfootbolist + *n) = footbolist0;
+    (*n)++;
+    return newfootbolist;
+}
+Footbolist* RemoveFootbolist(Footbolist* footbolist, int* n, Footbolist footbolist0) {
+    if (*n > 0) {
+        int k = FindFootbolist(footbolist, *n, footbolist0);
+        if (k == -1) {
+            cout << "Error! Requested footbolist can't be found" << endl;
+            return footbolist;
+        }
+        else {
+            int i;
+            Footbolist* newfootbolist = new Footbolist[(*n) - 1];
+            for (i = 0; i < k; i++) {
+                *(newfootbolist + i) = *(footbolist + i);
+            }
+            for (i = k + 1; i < *n; i++) {
+                *(newfootbolist + i - 1) = *(footbolist + i);
+            }
+            (*n)--;

```

Comment:
If you already form a new array, why just not have an one for with one if statement there for skip element of array number k